### PR TITLE
Fixing the log method to send only JSON content

### PR DESCRIPTION
**Pull request by ilonashub**
Following this issue, and the suggestion from this issue: https://github.com/shadabahmed/logstasher/issues/41
Changing the `log` method to write only 100% JSON logs, and changing the corresponding tests.

### CODE REVIEW

**Comment by shadabahmed** [comment]
Released Logstasher version 0.6.2 with this PR :)

Thanks a ton :+1:

**Comment by ilonashub** [comment]
Great, thanks!

On Mon, Jan 5, 2015 at 5:27 PM, Shadab Ahmed <notifications@github.com>
wrote:

> Released Logstasher version 0.6.2 with this PR :)
>
> Thanks a ton [image: :+1:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/shadabahmed/logstasher/pull/62#issuecomment-68722910>.
>



-- 

*Ilona Shub*
Software Developer
Settlement App
+972 54 7742427

Klarna
98 Yigal Alon st. Electra building. 13 floor.
Tel Aviv 67891 Israel
+972 54 8009 819
klarna.com

File: lib/logstasher.rb [hunk 1]
```diff
@@ -117,7 +117,7 @@ def custom_fields=(val)
   def log(severity, msg)
     if self.logger && self.logger.send("#{severity}?")
       event = LogStash::Event.new('@source' => self.source, '@fields' => {:message => msg, :level => severity}, '@tags' => ['log'])
-      self.logger.send severity, event.to_json
+      self.logger << event.to_json + "\n"
```
Comment (rabidscorpio) [inline lib/logstasher.rb]: I don't understand why this change was made, it completely breaks native json parsing in logstash without using the json_lines codec.

**Comment by shadabahmed** [comment]
hey @rabidscorpio - can you share the json where this is breaking ? I am a little out of touch but sure can try to parse why the change was required and what it is breaking
